client/blocks/comment-detail/comment-detail-post.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ReaderAvatar from 'blocks/reader-avatar';
+
+export const CommentDetailPost = ( {
+	author,
+	siteIcon,
+	title,
+	url,
+} ) =>
+	<div className="comment-detail__post">
+		<ReaderAvatar
```

Comment:
:D I wonder if we should rename the `ReaderAvatar` to something more generic in a future PR if we end up using this outside of the reader. cc @blowery